include/erlang_ls.hrl

```diff
@@ -149,10 +149,11 @@
 %%------------------------------------------------------------------------------
 %% Workspace Edit
 %%------------------------------------------------------------------------------
--type workspace_edit() :: #{ changes         => #{ binary() := [text_edit()]
-                                                 }
-                           , documentChanges => [text_document_edit()]
-                           }.
+
+-type document_change() :: [text_document_edit()].
+
+-type workspace_edit() :: #{ changes         := map() }
```

Comment:
Looking at the protocol, both fields are marked as _optional_ (with `changes` being ignored if `documentChanges` is available), so `=>` should be used in the type definition.
The `changes` should be a struct of `uri() => [text_edit()]` pairs, so the original definition looks correct to me (where `binary()` could be replaced with a more accurate `uri()`.